components/core/footer/Footer.vue

```diff
@@ -1,52 +1,70 @@
 <template>
-    <div class="flex flex-col px-4 py-8 bg-dark-primary text-golden">
-        <div class="flex justify-center">
-            <div class="flex flex-col mr-20">
-                <locale-link to="/about/code-of-conduct">
-                    {{ $t('codeOfConduct') }}
-                </locale-link>
-                <!-- 
-                <locale-link to="/about/staff">{{ $t('staff') }}</locale-link>
-                <locale-link to="/about/community">
-                    {{ $t('community') }}
-                </locale-link>
-                <locale-link to="/about/privacy-policy">
-                    {{ $t('privacyPolicy') }}
-                </locale-link>
-                -->
+    <footer
+        class="w-full flex flex-col justify-between bg-dark-secondary text-golden"
+    >
+        <div class="w-full p-10">
+            <div class="h-32 flex justify-center">
+                <div class="flex flex-col">
+                    <locale-link to="/about/code-of-conduct">
+                        {{ $t('codeOfConduct') }}
+                    </locale-link>
+                    <!--
+                    <locale-link to="/about/staff">{{ $t('staff') }}</locale-link>
+                    <locale-link to="/about/community">
+                        {{ $t('community') }}
+                    </locale-link>
+                    <locale-link to="/about/privacy-policy">
+                        {{ $t('privacyPolicy') }}
+                    </locale-link>
+                    -->
+                </div>
             </div>
-            <div class="flex flex-col ml-20">
-                <ext-link href="https://pycontw.blogspot.tw/">
-                    Blogspot
-                </ext-link>
-                <ext-link href="https://www.facebook.com/pycontw">
-                    Facebook
-                </ext-link>
-                <ext-link href="https://twitter.com/pycontw">Twitter</ext-link>
-                <a
+            <div class="w-full flex justify-center items-center flex-wrap">
+                <footer-icon
+                    href="https://pycontw.blogspot.tw/"
+                    :icon="['fab', 'blogger']"
+                />
+                <footer-icon
+                    href="https://www/facebook.com/pycontw"
+                    :icon="['fab', 'facebook']"
+                />
+                <footer-icon href="" :icon="['fab', 'telegram']" />
```

Comment:
Add the invitaion link of telegram channel.
```suggestion
                <footer-icon 
                    href="https://t.me/pycontw_chat" 
                    :icon="['fab', 'telegram']" 
                />
```

Reply:
updated